src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,83 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] text = openFile(fromFileName);
+        int[] data = calcutData(text);
+        String[] dataForPrint = printData(data);
+        putDataToFile(dataForPrint,toFileName);
+    }
+
+    private String[] openFile(String fileName) {
+        StringBuilder string = new StringBuilder();
+        try {
+            File file = new File(fileName);
+            FileReader fileReader = new FileReader(file);
+            BufferedReader bufferedReader = new BufferedReader(fileReader);
+            String text = bufferedReader.readLine();
+            while (text != null) {
+                string.append(text).append("///");
+                text = bufferedReader.readLine();
+            }
+            bufferedReader.close();
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t open file!", e);
+        }
+        if (string.toString().isEmpty()) {
+            String[] empty = {};
+            return empty;
+        } else {
+            return string.toString().split("///");
+        }
+    }
+
+    private int[] calcutData(String[] str) {
+        int[] result = new int[]{0,0};
+        for (int i = 0; i < str.length; i++) {
+            String[] strings = str[i].split(",");
+            if (strings[0].equals("supply")) {
+                result[0] += Integer.parseInt(strings[1]);
```

Comment:
0 and 1 should be made constants because it should be more clear what is going on